package.json

```diff
@@ -165,16 +165,16 @@
     "@visx/responsive": "^2.10.0",
     "@visx/shape": "^2.11.1",
     "@walletconnect/ethereum-provider": "^1.8.0",
-    "@web3-react/coinbase-wallet": "8.0.35-beta.0",
-    "@web3-react/core": "8.0.35-beta.0",
-    "@web3-react/eip1193": "8.0.27-beta.0",
-    "@web3-react/empty": "8.0.20-beta.0",
-    "@web3-react/gnosis-safe": "8.0.7-beta.0",
-    "@web3-react/metamask": "8.0.30-beta.0",
-    "@web3-react/network": "8.0.27-beta.0",
-    "@web3-react/types": "8.0.20-beta.0",
-    "@web3-react/url": "8.0.25-beta.0",
-    "@web3-react/walletconnect": "8.0.37-beta.0",
+    "@web3-react/coinbase-wallet": "8.1.1-beta.0",
+    "@web3-react/core": "8.1.1-beta.0",
+    "@web3-react/eip1193": "8.1.1-beta.0",
+    "@web3-react/empty": "8.1.1-beta.0",
+    "@web3-react/gnosis-safe": "8.1.1-beta.0",
+    "@web3-react/metamask": "8.1.1-beta.0",
+    "@web3-react/network": "8.1.1-beta.0",
+    "@web3-react/types": "8.1.1-beta.0",
+    "@web3-react/url": "8.1.1-beta.0",
+    "@web3-react/walletconnect": "8.1.1-beta.0",
```

Comment:
Can we just remove these and use the dependencies brought in by @uniswap/widgets? That way they will stay in sync.

Reply:
that seems a bit too implicit for me... what if we used resolutions?

Reply:
Agreed. We're using this directly in this project, so makes sense to have an explicit dependency.